test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +35,47 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsAreNull() throws Exception {
+        // empty list
+        assertNotNull();
+
+        // one null object
+        assertHasNull((Object) null);
+
+        // list of null and non-null objects
+        assertHasNull("abc", null, "def");
+        assertHasNull(1, 2, 3, null, 5);
+        assertHasNull(null, "abc");
+        assertHasNull(null, 123);
+        assertHasNull(null, null, "abc");
+        assertHasNull(null, null, 123);
+        assertHasNull("abc", 123, null, 456, "def");
+        assertHasNull("abc", "def", 123, 456, null, null);
+
+        // list of all null objects
+        assertHasNull(null, null);
+        assertHasNull(null, null, null);
+
+        // list of no null objects
+        assertNotNull("abc", "ab", "a");
```

Comment:
Logical ordering will be: 

- empty list
- no null
- some null 
- all null

So I think you can move this section of `no nulls` above